bench-vortex/src/bin/tpch_benchmark.rs

```diff
@@ -25,14 +28,26 @@ struct Args {
     exclude_queries: Option<Vec<usize>>,
     #[arg(short, long)]
     threads: Option<usize>,
+    #[arg(long)]
+    use_remote_data_dir: Option<String>,
     #[arg(short, long, default_value_t = true, default_missing_value = "true", action = ArgAction::Set)]
     warmup: bool,
     #[arg(short, long, default_value = "5")]
     iterations: usize,
-    #[arg(long)]
-    only_vortex: bool,
+    #[arg(long, value_delimiter = ',')]
+    formats: Option<Vec<String>>,
+    #[arg(long, default_value_t = 1)]
+    scale_factor: u8,
     #[arg(short, long)]
-    verbose: bool,
+    only_vortex: bool,
+    /// Allow specifying verbosity of logging.
+    ///
+    /// None: WARN
+    /// -v:   INFO
+    /// -vv:  DEBUG
+    /// -vvv: TRACE
+    #[arg(short, action = ArgAction::Count)]
+    verbosity: u8,
```

Comment:
seems like we are building similar things
1. we should standardize the arguments for benchmarks as much as we can
2. just setting `RUST_LOG` isn't that bad

Reply:
I'm ripping out all my changes to adopt yours.